File: Python3/프로그래머스/0/181882. 조건에 맞게 수열 변환하기 1/조건에 맞게 수열 변환하기 1.py
def solution(arr):
    answer = []
    
    for index in range(len(arr)):
        num = arr[index]
        if num >= 50 and num % 2 == 0:
            num /= 2
        elif num < 50 and num % 2 != 0:
            num *= 2
        
        answer.append(num)
        
    return answer

File: Swift/프로그래머스/0/181882. 조건에 맞게 수열 변환하기 1/조건에 맞게 수열 변환하기 1.swift
import Foundation

func solution(_ arr:[Int]) -> [Int] {
    var answer: [Int] = []
    for index in 0..<arr.count {
        
        var num = arr[index]
        
        if num >= 50 && num % 2 == 0 {
            num = num / 2
        } else if num < 50 && num % 2 != 0 {
            num = num * 2
        }
        answer.append(num)
    }
    
    return answer
}